084-open-feature-integration.md

```diff
@@ -0,0 +1,477 @@
+# OpenFeature Integration to Strimzi
+
+Proposal to integrate OpenFeature into Strimzi to enhance feature management capabilities.
+The integration of OpenFeature across all operators will be streamlined, thanks to the enhancements proposed in [#74](https://github.com/strimzi/proposals/pull/118).
+
+## Current Situation
+Strimzi, currently leverages environment variables for feature management. While this approach is effective, it requires rolling updates for any changes, which can lead to service downtime. 
+Moreover, it lacks the flexibility offered by modern feature flag systems, which can provide dynamic control with zero downtime when toggling features.
+
+## Motivation
+
+To provide Strimzi users with dynamic control over features without needing service restarts, enhancing the efficiency and adaptability of their deployments.
+
+## Proposal
+
+This proposal suggests the integration of OpenFeature, providing two primary methods for feature management:
+
+For the current users (i.e., having classic ENV variable STRIMZI_FEATURE_GATES), we will use [Enviroment Variable Provider](https://github.com/open-feature/java-sdk-contrib/tree/main/providers/env-var)
+already implemented by OpenFeature folks. Some of the common characteristics of such provider are:
+1. Default method currently used. 
+2. Feature gates are set via environment variables.
+3. Requires rolling updates for changes.
```

Comment:
You link to an existing provider. But how does it provide the backwards compatibility for how we configure the feature gates?

Reply:
Simply by using just `STRIMZI_FEATURE_GATES` env var in all operators as we do it currently.